api/apps/prompt_library_app.py

```
@@ -0,0 +1,391 @@
+    admin_required,
+    bad_request_response,
+    log_activity,
+    not_found_response,
+    server_error_response,
+    validate_request,
+    validate_sort_order,
+)
+from api import settings
+from api.utils.api_utils import get_json_result
+
+@manager.route("/create-prompt", methods=["POST"])
```

Ruff (0.8.2)

Undefined name `manager`

(F821)